launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java

```diff
@@ -95,7 +95,8 @@ public boolean isFinal() {
   void kill();
 
   /**
-   * Disconnects the handle from the application, without stopping it. After this method is called,
+   * Disconnects the handle from the application. If using {@link SparkLauncher#autoShutdown()}
```

Comment:
It looks the documentation generation for Javadoc 8 is being failed due to this line - 

```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/launcher/src/main/java/org/apache/spark/launcher/SparkAppHandle.java:98: error: reference not found
[error]    * Disconnects the handle from the application. If using {@link SparkLauncher#autoShutdown()}
[error]                                                                   ^
```

Probably, wrap it with `` `... ` `` as I did before - https://github.com/apache/spark/pull/16013 or find a way to make the link properly.

The other errors seem spurious. Please refer my observation - https://github.com/apache/spark/pull/17389#issuecomment-288438704